base/shell/explorer/settings.cpp

```diff
@@ -32,6 +32,7 @@ BOOL TaskbarSettings::Save()
     SHSetValueW(hkExplorer, L"Advanced", L"TaskbarSizeMove", REG_DWORD, &bAllowSizeMove, sizeof(bAllowSizeMove));
     sr.cbSize = sizeof(sr);
     SHSetValueW(hkExplorer, L"StuckRects2", L"Settings", REG_BINARY, &sr, sizeof(sr));
+    SHSetValueW(hkExplorer, L"Advanced", L"TaskbarSmallIcons", REG_DWORD, &bSmallIcons, sizeof(bSmallIcons));
```

Comment:
I experimented on my XPSP3 and set DWORD TaskbarSmallIcons to 0 in the registry, and then restarted the explorer.
It did not cause the icons to be larger.
Does that mean XPSP3 would not support the larger icons yet, and that this is a Vista+ feature?
Or does that mean you chose a different name in the registry than MS Windows for this setting?

In case you chose a different setting than MS, then please harmonize with MS.